lib/js-api/src/alfrescoApi.ts

```diff
@@ -637,4 +632,24 @@ export class AlfrescoApi implements Emitter, AlfrescoApiType {
             this.on(event, callback);
         }
     }
+
+    private addPromiseListeners<T = void>(promise: Promise<T>, eventEmitter: typeof ee): Promise<T> {
```

Comment:
Do we have two methods with the same name one private and the other public? https://github.com/Alfresco/alfresco-ng2-components/pull/10646/files#diff-fe3a8c74738d039da3c7055ca74fbb6c79cdad479ce82777e19f0eb0524d670cR367